js/notifications.js

```diff
@@ -12,38 +12,27 @@
         MESSAGE : 'message'
     };
 
+    var enabled = false;
```

Comment:
Why not make this a member of `Whisper.Notifications`? Set `this.enabled = false` in `initialize()` below?

Reply:
Hmm guess I was just being paranoid about allowing the state to be modified directly rather than through the explicit enable/disable functions. :woman_shrugging: